.circleci/config.yml

```diff
@@ -70,6 +80,17 @@ install_dep_17_cpu: &install_dep_17_cpu
         python -c 'import torch; print("Torch version:", torch.__version__)'
         python -m torch.utils.collect_env
 
+install_dep_17_cpu_cu11: &install_dep_17_cpu_cu11
+  - run:
+      name: Install Dependencies
+      command: |
+        sudo apt-get install -y libopenmpi-dev
+        pip install --progress-bar off torch==1.7.1+cu110 -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
I'm not seeing where cuda 11 is actually requested on the agent, bar that this will run on cpu only I think

Reply:
given the other PR (thanks for waiting and sorry for the merge conflicts !) I think you'll probably get something good with just installing requirements if you want 1.7 on cpu, no need to install a specific version, at least that's my guess

Reply:
No problem. Thanks for fixing the other issue. That will make this PR even better. I will handle the conflicts.

Yeah for cpu, we can just install the version without cuda. I will test it out.

Reply:
To support CUDA 11.0, we would need to create a docker image with the correct kernel driver.